pkg/cmd/util/tokencmd/challenging_client.go

```diff
@@ -27,6 +27,9 @@ var basicAuthRegex = regexp.MustCompile(basicAuthPattern)
 
 // Do watches for unauthorized challenges.  If we know to respond, we respond to the challenge
 func (client *challengingClient) Do(req *http.Request) (*http.Response, error) {
+	// Set custom header required by server to avoid CSRF attacks on browsers using basic auth
+	req.Header.Set("X-CSRF-Token", "1")
```

Comment:
Third usage.  Two constants (so we don't tie the packages together, unless we already have an "ours" package) and a pointer to keep them in sync.